### DISCUSSION TITLE
Play (from, to)

### DISCUSSION THREAD

**Discussion by Binah-CS** [katspaugh/wavesurfer.js#3587] [Q&A]
Can I play a specific timeframe of audio directly, not using Region?
I can setTime - for the start point, but how to I set the end point?

**Comment by katspaugh** [comment]
Atm you can only do this with the WebAudioPlayer like this:

https://wavesurfer.xyz/examples/?998f3697a6ce83671926c2f3868d9b7d

**Comment by Binah-CS** [comment]
I did it by creating a temporary region.
Thought I might be missing something.

Thank you.

Still the absolutely BEST audio player ....
